packages/react-components/src/components/styles/TypingIndicator.styles.ts

```diff
@@ -7,7 +7,9 @@ import { mergeStyles } from '@fluentui/react';
  * @private
  */
 export const typingIndicatorContainerStyle = mergeStyles({
-  minHeight: '2.125rem'
+  minHeight: '2.125rem',
+  // flexFlow set to column-reverse to align the text to the bottom of the container
+  flexFlow: 'column-reverse'
```

Comment:
because flex boxes have a wide variety of css properties that are hard to dissociate from each other (and because when changing css its incredibly hard to ensure no other properties are affected) we want to avoid writing css as much as possible.

Instead we want to use a semantic driven approach and use Fluent's [Stack](https://developer.microsoft.com/en-us/fluentui#/controls/web/stack)
These have properties like `horiztonal` and `align=end` to avoid having to write css and have a semantical-driven approach to how components are laid out that very readable&understandable.

Instead can you do something like:
![image](https://user-images.githubusercontent.com/2684369/149232159-ae3f28b5-e5db-4112-9b3b-d449384697c7.png)
? (not sure if the alignment should be 'baseline' or 'end')

If not keep this as is!

Reply:
flex layout is taking precedence over the properties thus vertical align is not working in this case. The other property that we can change is justifyContent='end' which is again property to flex layout. So, I am going to use column-reverse in this case.